packages/core/test/sources/console-child/truffle-config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  networks: {
+    funTimeNetwork: {
+      host: "127.0.0.1",
+      port: 5555,
+      network_id: "*",
+      gas: 5500000,
+      confirmations: 2
+    },
+    crazyTimeNetwork: {
```

Comment:
What a crrraaaaazy network!

Reply:
I guess it is not necessary. I will remove the `gas` property. Thanks @eggplantzzz.

Reply:
Oh, I guess that stuff would be useful if you wanted to test that user-specified parameters end up in the return for this function.

Reply:
ah, this is related to my comment as well. You could stick a `customUserProperty` in the network config and assert it shows up in the verification step.

Reply:
Yeah! I agree with you @cds-amal. I will add a `customUserProperty` in the `network` config. Thanks for your suggestions @cds-amal and @eggplantzzz.